get-random-throw.js

```diff
@@ -0,0 +1,6 @@
+function getRandomThrow() {
+    const getRandom = Math.ceil(Math.random() * 3);
+    return getRandom;
+}
```

Comment:
this function is a great place to do the logic where you assign the output of getRandom to a rock, paper, or scissors string.